sw/device/tests/dif/dif_alert_handler_unittest.cc

```diff
@@ -808,28 +812,28 @@ TEST_F(CauseTest, NullArgs) {
 
 class EscalationTest : public AlertTest {};
 
-TEST_F(EscalationTest, CanClear) {
-  bool flag;
-
-  EXPECT_READ32(ALERT_HANDLER_CLASSB_REGWEN_REG_OFFSET, true);
-  EXPECT_EQ(dif_alert_handler_escalation_can_clear(
-                &handler_, kDifAlertHandlerClassB, &flag),
-            kDifAlertHandlerOk);
-  EXPECT_TRUE(flag);
-
-  EXPECT_READ32(ALERT_HANDLER_CLASSA_REGWEN_REG_OFFSET, false);
-  EXPECT_EQ(dif_alert_handler_escalation_can_clear(
-                &handler_, kDifAlertHandlerClassA, &flag),
-            kDifAlertHandlerOk);
-  EXPECT_FALSE(flag);
-}
-
-TEST_F(EscalationTest, Disable) {
-  EXPECT_WRITE32(ALERT_HANDLER_CLASSC_REGWEN_REG_OFFSET, true);
-  EXPECT_EQ(dif_alert_handler_escalation_disable_clearing(
-                &handler_, kDifAlertHandlerClassC),
-            kDifAlertHandlerOk);
-}
+// TEST_F(EscalationTest, CanClear) {
+//   bool flag;
+
+//   EXPECT_READ32(ALERT_HANDLER_CLASSB_REGWEN_REG_OFFSET, true);
```

Comment:
ALERT_HANDLER_CLASSB_ **CLR** _REGWEN_REG_OFFSET

And similar for L824 and L832.

Reply:
Thanks! I have re-enabled those tests again, since they work now with this change.